data/remnant/remnant 2 side missions.txt

```diff
@@ -144,3 +144,201 @@ mission "Remnant: Visit to Paradise 1"
 			label mainreturn
 			`	Taely leads you around the area for a few more hours, pointing out historical features, places where they resurrected trades and skills from before the advent of flight, and other interesting things. Eventually she heads back to the train station, which deposits you back in the main spaceport.`
 			`	"I hope you enjoyed the tour, <first>. It was interesting to see our history through fresh eyes." She smiles micheviously. "And I enjoyed the hoverboarding, too. Maybe we can do it again sometime." She pauses for a moment, focusing on something you cannot see. "Looks like duty calls. Embers guide your way, Captain <last>." She waves, and heads off into the crowd.`
+
+
+
+mission "Remnant: Celebration 1"
+	name "Deliver Celebration records to <destination>"
+	description "The Remnant have asked you to take records of their celebration to <destination>."
+	minor
+	source
+		government "Remnant"
+	destination "Ssil Vida"
+	to offer
+		has "event: remnant: cognizance calm"
+		has "license: Remnant Capital"
+		random < 50
+		"day" = 7
+	on offer
+		conversation
+			`You stop by the starport just in time to see a large group of Remnant departing. A few of the nearest notice your interest and begin gesturing recognition or various invitations to accompany them, but before you are able to respond, Taely appears from the stream of people and approaches you.`
+			`	"Well met, <first>. We were just going to the monthly Celebration of Remembrance. Would you like to accompany us?"`
+			choice
+				`	"Yes, I would."`
+					goto celebrationattendance
+				`	"What is that?"`
+					goto celebrationdefinition
+				`	"No thanks."`
+			label celebrationdeclined
+			`	She nods. "Well, may the Embers guide you, <first>." With that, she rejoins the stream of people as they vanish from sight.`
```

Comment:
Hmm, that makes it sound more like Taely vanishing, as opposed to the stream.

Reply:
It would, but that's a change I made when I switched the wording. Thought it would help address the awkwardness @quyykk mentioned to talk about when you could no longer see the group versus when you could no longer see her.

But suggestion from @TheMarksman-ES is improvement on original even though switching that back.

Reply:
Would it be acceptable to just leave it as-is, then?

Reply:
Alright then, leaving it as-is. Thanks!